gcc/rust/ast/rust-item.h

```diff
@@ -3548,6 +3548,8 @@ class NamedFunctionParam
    * '_'). */
   bool has_name () const { return name != "_" && name != ""; }
 
+  bool has_type () const { return param_type != nullptr; }
```

Comment:
If a `NamedFunctionParam` cannot have a type then we should probably wrap the type pointer within an optional.